### PR TITLE
feat(company wallet): update company wallet end point

### PR DESCRIPTION
**Pull request by manojava-gk**
## Description

add new endpoint from ssi credential base
update the response schema. handle error and not valid membership wallet case in the ui.

## Why

new change request

## Issue

#621 

## Checklist

Please delete options that are not relevant.

- [x] I have followed the [contributing guidelines](https://github.com/eclipse-tractusx/portal-assets/blob/main/docs/developer/Technical%20Documentation/Dev%20Process/How%20to%20contribute.md#commit-and-pr-guidelines)
- [ ] I have performed [IP checks](https://eclipse-tractusx.github.io/docs/release/trg-7/trg-7-04#checking-libraries-using-the-eclipse-dash-license-tool) for added or updated 3rd party libraries
- [ ] I have created and linked IP issues or requested their creation by a committer
- [x] I have performed a self-review of my own code
- [x] I have successfully tested my changes locally
- [ ] I have added tests that prove my changes work
- [ ] I have checked that new and existing tests pass locally with my changes
- [ ] I have commented my code, particularly in hard-to-understand areas

### CODE REVIEW

File: src/utils/rtkUtil.ts [hunk 1]
```diff
@@ -59,6 +60,14 @@ export const apiIdentityWalletQuery = () => ({
   },
 })
 
+export const apiSSICredentialQuery = () => ({
```
Comment (nidhigarg-bmw) [inline src/utils/rtkUtil.ts:63]: @manojava-gk I already created the function for SsiCredential on line 39. Please use that one. If not using anywhere then can you please remove that one.

Comment (manojava-gk) [reply src/utils/rtkUtil.ts]: removed. Thanks

File: src/components/pages/CompanyWallet/index.tsx [hunk 2]
```diff
@@ -33,28 +33,27 @@ import { groupBy } from 'lodash'
...
-      item.credentialSubject[0].type === CredentialSubjectType.Framework
+  const groupedItems = groupBy(data, (item: WalletContent) =>
+    item.credentialType === CredentialType.TRACEABILITY_FRAMEWORK
+      ? item.credentialType
+      : 'Others'
   )
 
+  console.log(groupedItems)
+
   return (
```
Comment (lavanya-bmw) [inline src/components/pages/CompanyWallet/index.tsx:57]: Please remove console log

File: src/components/pages/CompanyWallet/index.tsx [hunk 3]
```diff
@@ -33,28 +33,27 @@ import { groupBy } from 'lodash'
...
 
-  const groupedItems = groupBy(
-    data?.content,
-    (item: WalletContent) =>
-      item.credentialSubject[0].type === CredentialSubjectType.CompanyRole ||
-      item.credentialSubject[0].type === CredentialSubjectType.Framework
+  const groupedItems = groupBy(data, (item: WalletContent) =>
+    item.credentialType === CredentialType.TRACEABILITY_FRAMEWORK
+      ? item.credentialType
+      : 'Others'
```
Comment (lavanya-bmw) [inline src/components/pages/CompanyWallet/index.tsx:52]: Please remove the hardcoded string

**Review by lavanya-bmw — CHANGES_REQUESTED** [review]

**Comment by manojava-gk** [comment]
@oyo @evegufy zip file shows as 140 bytes but after unzipping it is showing as zero bytes and file is empty

Screenshot 2024-05-10 at 3 32 01 PM

**Comment by manojava-gk** [comment]
https://github.com/eclipse-tractusx/portal-frontend/actions/runs/9029891241

**Comment by evegufy** [comment]
> @oyo @evegufy zip file shows as 140 bytes but after unzipping it is showing as zero bytes and file is empty
> 
> Screenshot 2024-05-10 at 3 32 01 PM

as discussed, sometimes the the dash tool service is temporarily unavailable and should be retriggered after some time passing. I just retriggered it manually, and the check runs on green.

File: src/assets/locales/de/main.json [hunk 4]
```diff
@@ -1879,6 +1879,8 @@
       "description": "Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.",
       "activate": "Connected",
       "activateDescription": "Sie haben Ihr Catena-X-Wallet aktiviert.",
+      "error": "Not Connected",
+      "errorDescription": "Wallet-Mieter nicht gefunden",
```
[the remark this answers is not in the document]
Comment (manojava-gk) [reply src/assets/locales/de/main.json:1883]: I have picked it from google translation

File: src/components/pages/CompanyWallet/WalletCard.tsx [hunk 5]
```diff
@@ -29,52 +28,86 @@ import './CompanyWallet.scss'
...
-                </div>
-              )}
+              <img
+                src="/cx-logo.svg"
+                alt="cx logo"
+                style={{
+                  width: 40,
+                }}
+              />
             </div>
```
[the remark this answers is not in the document]
Comment (manojava-gk) [reply src/components/pages/CompanyWallet/WalletCard.tsx:106]: This section has both row and column flex property. So it is wrapped in a div

File: src/assets/locales/de/main.json (hunk 4 above)
[the remark this answers is not in the document]
Comment (jjeroch) [reply src/assets/locales/de/main.json]: updated

**Review by evegufy — APPROVED** [review]

**Pull request merged**